components/AlertTItle/AlertTitle.test.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { AlertTitle } from './AlertTitle';
+
+describe('AlertTitle', () => {
+  it('renders the title text', () => {
+    const { getByText } = render(<AlertTitle title="Test Title" />);
```

Comment:
Suggestion: I do not believe the const {getByText} is necessary. What we normally do is just add a data-testid to the actual code. Leave the render part, but just add another line saying:

const alertTitle = screen.getByTestId('whatever');

Then you can just: expect(alertTitle).

Edit: my assumption is you kind of are already giving it a title to render but the actual component takes in a prop